content/starter-configs/examples/adobe-experience-manager.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Adobe Experience Manager"
+date: Tue, Mar 23 2021
+weight: 7
+---
+
+This AEM config example starts with a Tugboat Debian image, installs Java, and launches both the author and publisher
+instances. You'll need to do some customizing for your app, but this should get you started.
+
+## Configure Tugboat
+
+The Tugboat configuration is managed by a [YAML file](/setting-up-tugboat/create-a-tugboat-config-file/) at
+`.tugboat/config.yml` in your git repository. For the full config file, along with installation instructions, please see
+the corresponding GitHub repository for
+[configuring Adobe Experience Manager on Tugboat](https://github.com/TugboatQA/aem).
+
+```yaml
+services:
+  author:
+    image: tugboatqa/debian:10
+    default: true
+    expose: 4502
+
+    commands:
+      init:
+        # Install Java SDK and Maven.
+        - apt-get update && apt-get install -y openjdk-11-jdk maven
+
+        # Create directory structure. license.properties is base64 encoded as a TB repos variable.
+        - mkdir -p /opt/aem/author
+        - echo "${LICENSE_PROPERTIES}" | base64 --decode > /opt/aem/author/license.properties
+        - ln -snf ${TUGBOAT_ROOT} /opt/aem/code
+
+        # Download AEM quickstart file and install
+        - wget -O /opt/aem/author/aem-author-p4502.jar "${AEM_FILE_URL}"
+        - cd /opt/aem/author && java -jar aem-author-p4502.jar -unpack
+
+        # Configure Adobe public profile.
+        - cp -r ${TUGBOAT_ROOT}/.tugboat/resources/.m2 ~/.m2
+        - mvn help:effective-settings
+
+        # Start aem for the first time and wait for a 200, authenticated response
+        - CQ_PORT=4502 /opt/aem/author/crx-quickstart/bin/start
+        - while [ "$(curl -u admin:admin --head --location --connect-timeout 5 -s -o /dev/null -w ''%{http_code}''
```

Comment:
We might be able to do `wget -O /dev/null -o /dev/null -t 10 --retry-connrefused http://admin:admin@localhost:4502`